tests/test_mlip_calculators.py

```diff
@@ -131,7 +134,7 @@ def test_invalid_arch():
 def test_invalid_model_path(arch, model_path):
     """Test error raised for invalid model_path."""
     skip_extras(arch)
-    with pytest.raises((ValueError, RuntimeError, KeyError)):
+    with pytest.raises((ValueError, RuntimeError, KeyError, AssertionError)):
```

Comment:
Ok, I'm thinking this wants to be a bit more specific, I think adding a `pytest.raises` (hopefully with `match`) into the `parametrize` isn't asking too much. 

Probably for another PR.